config/routes.rb

```diff
@@ -3,5 +3,8 @@
   root to: 'games#index' # our landing page with selection of games
   resources :instances, only: [:create, :show, :update] do
     resources :player_inputs, only: [:create, :index]
+    collection do
+      post :join # to join the instance via pin
+    end
   end
 end
```

Comment:
🤔 Goal of the route in the end is to create a player. I think we may as well create `resources :player, only: [:create]` below like 9. In the future this will also be handy to have a destroy player if ever we decide to allow the player to leave a game instance before the game starts.